src/Doctrine/EventSubscriber/PostSoftDeleteSubscriber.php

```diff
@@ -89,5 +91,15 @@ public function postSoftDelete(LifecycleEventArgs $args)
 
             $unitOfWork->computeChangeSets();
         }
+
+        if ($entity instanceof Store) {
```

Comment:
Maybe the store should be also "reset" here. I mean, we should set the pricing rule set, time slots & packages to NULL. 
I'm foreseeing the deletion of pricing rules, time slots, packages. This way, if soft deleted stores are already not using them anymore, it would simplify things.

Reply:
OK will do in the next PRs